Core/SessionHelper.php

```diff
@@ -70,15 +70,17 @@ public static function getBic()
     /**
      * Returns date of birth
      *
+     * @param string $sPaymentMethodName
+     *
      * @return string date of birth formatted for db (format 'Y-m-d')
      *
      * @since 1.2.0
      */
-    public static function getDbDateOfBirth()
+    public static function getDbDateOfBirth($sPaymentMethodName)
```

Comment:
Every method now expects '$sPaymentMethodName', is it not better to make a constructor for the payment method name and then, use it inside of the class.

Reply:
this method is inside of the SessionHelper, It's not possible to make a constructor for the payment method name in here